src/components/message-box/MessageBox.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Box, {COLOR} from '../box/Box';
+import Icon, {TYPE as iconTypes, ICON_COLOR} from '../icons/Icon';
+
+export {COLOR};
+
+const MessageBox = ({children, onClose, ...props}) => {
+  const boxClass = classNames({
+    'sg-box--with-close': onClose
+  });
+
+  return (
+    <div className={'sg-message-box'}>
+      {onClose ?
+        <div className="sg-message-box__close" onClose={onClose}>
+          <Icon type={iconTypes.X} color={ICON_COLOR.DARK} size={14} />
+        </div> : null
+      }
+      <Box {...props} className={boxClass} >
```

Comment:
shouldn't we have specific padding for Box ?

Reply:
default one is ok according to designers

Reply:
i can set it only for small or large (there is no normal) so i would mess it up only